imap/global.c

```diff
@@ -346,6 +346,8 @@ EXPORTED int cyrus_init(const char *alt_config, const char *ident, unsigned flag
                                   config_getswitch(IMAPOPT_UNIX_GROUP_ENABLE));
         libcyrus_config_setswitch(CYRUSOPT_USERNAME_TOLOWER,
                                   config_getswitch(IMAPOPT_USERNAME_TOLOWER));
+	libcyrus_config_setswitch(CYRUSOPT_NORMALIZEUID,
+				  config_getswitch(CYRUSOPT_NORMALIZEUID));
```

Comment:
Bad indentation.

Reply:
The bad indentation can be fixed easily enough with tools/remove-tabs.pl...

The bigger problem here is calling `config_getswitch()` with a CYRUSOPT_ option rather than an IMAPOPT_ one.  It should probably be `config_getswitch(IMAPOPT_NORMALIZEUID)` like the other lines around it.

I think this is the source of one of the warnings @anatoli26 noticed.